test/manual/amp-lightbox-gallery-launch.amp.html

```diff
@@ -14,13 +14,23 @@
   <script async custom-element="amp-instagram" src="https://cdn.ampproject.org/v0/amp-instagram-0.1.js"></script>
   <script async custom-element="amp-facebook" src="https://cdn.ampproject.org/v0/amp-facebook-0.1.js"></script>
   <script async custom-element="amp-ad" src="https://cdn.ampproject.org/v0/amp-ad-0.1.js"></script>
+  <script async custom-element="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.1.js"></script>
+  <script async custom-element="amp-analytics" src="https://cdn.ampproject.org/v0/amp-analytics-0.1.js"></script>
 
   <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
   <style amp-custom>
     body {
       font-family: sans-serif;
     }
 
+    /* This is necessary to fix styling for amp-youtube and amp-video */
```

Comment:
@aghassemi @cvializ how do we feel about setting `i-amphtml-slide-item > *` to `height: auto` in `amp-carousel.css`?

Reply:
Hmm should this change only apply to these elements when the carousel is inside the lightbox gallery? Why do only these need to be updated here?

Reply:
Potentially a risky change. Given this has come up at least 3 times now, we should keep https://github.com/ampproject/amphtml/issues/13677 open and make the change there but we should:
1- do some extensive manual testing
2- put it behind an experiment and ramp up slowly

Reply:
I don't mind moving this to `amp-lightbox-gallery` CSS file until #13677 is fixed (with some comments about the related issue)